README.md

```diff
@@ -2,23 +2,29 @@
 
 # OpenTracing JAX-RS Instrumentation
 
-OpenTracing instrumentation for JAX-RS standard. It supports server and client request tracing.
+OpenTracing instrumentation for JAX-RS standard. It supports tracing of server and client requests.
 
-Instrumentation by default adds set of standard tags and sets span operation name with HTTP method. 
-This can be overridden by span decorators.
+Instrumentation by default adds a set of standard HTTP tags and as an operation name it uses a string defined in `@Path` annotation. Custom tags or operation name can be defined in span decorators.
 
 ## Tracing Server Requests
+By default OpenTracing dynamic feature is automatically discovered and registered. The only needed configuration is to register tracer instance: `GlobalTracer.register(tracer)` at application startup.
```

Comment:
"The only configuration that is required is to register a tracer instance....." - although could also mention about the new TracerResolver contrib mechanism as providing a zero code option.